src/Core/Analysis/QuickFix.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SonarLint.VisualStudio.Core.Analysis
+{
+    public class QuickFix : IQuickFix
+    {
+        private static readonly IReadOnlyList<IEdit> emptyEdits = new List<IEdit>();
+
+        public QuickFix(string message, IReadOnlyList<IEdit> edits)
+        {
+            Message = message;
+            Edits = edits ?? emptyEdits;
```

Comment:
I think it can't happen, a fix has to have at least 1 edit so this collection should never be null